CHANGELOG.md

```diff
@@ -2,11 +2,26 @@
 
 ## unreleased
 
+### Node API changes
+
```

Comment:
While the name of the RPC is descriptive, I think that its useful to say what the RPC does exactly. In this case, `validateresource` will validate Handshake Resource JSON and will return an error message on an invalid Resource